src/app/lib/config/services/bengali.js

```diff
@@ -148,10 +152,6 @@ const service = {
         href: 'https://www.bbc.com/terms',
         text: 'Terms of Use',
       },
-      {
```

Comment:
Are we supposed to be removing these links?

Reply:
Yes, I checked with Product and they are not needed for any of the World Service sites.

Reply:
Brill, thanks for the follow up.